### DISCUSSION TITLE
Library suggestion for video interface

### DISCUSSION THREAD

**Discussion by KratosAurion400** [lamemakes/pilot-drive#30] [Ideas]
Have you considered looking into PYAV for audio and video integration with the head unit? Not sure if it'd be possible, but I'll see if I can fork my own version and see what I can accomplish. https://github.com/PyAV-Org/PyAV

But if you want to take a crack at seeing if that's an option, by all means.

**Comment by lamemakes** [comment]
I haven't considered this! I haven't had time to push development forward as much as I've wanted to lately, so if this is something that you'd be interested please fork and see where you get! I'd be happy to merge it if it proved fruitful.

Thanks for the inquiry - excited to hear the outcome. Cheers!
